templates/org/team/teams.tmpl

```diff
@@ -32,9 +32,7 @@
 					</div>
 					<div class="ui attached segment members">
 						{{range .Members}}
-							<a href="{{.HomeLink}}" title="{{.Name}}">
```

Comment:
Hmm, the new template doesn't set the name as title like previously. Should we adapt the template for that?

Reply:
the `{{avatar .}}` sets the title.